packages/app-desktop/gui/NoteSearchBar.jsx

```diff
@@ -164,12 +167,17 @@ class NoteSearchBarComponent extends React.Component {
 						onKeyDown={this.searchInput_keyDown}
 						ref="searchInput"
 						type="text"
-						style={{ width: 200, marginRight: 5, backgroundColor: this.backgroundColor, color: theme.color }}
+						style={{
+							width: 200, marginRight: 5,
+							backgroundColor: this.backgroundColor,
+							color: theme.color,
+						}}
 					/>
 					{allowScrolling ? nextButton : null}
 					{allowScrolling ? previousButton : null}
 					{allowScrolling ? matchesFoundString : null}
 					{!allowScrolling ? viewerWarning : null}
+					{this.isSearchResultFound === false && <span> Phrase Not Found </span>}
```

Comment:
You can see an example of how to localize text [here](https://github.com/laurent22/joplin/blob/4dbf7e2d38462b11c150f9bbcefdb47306ed9f05/packages/app-desktop/gui/NoteSearchBar.jsx#L164). Basically you just need to wrap the string with _(), so "Phrase not found" would be _("Phrase not found"). The Joplin localization tools take care of the rest.